File: Templates/TypeEditor/src/Templates/Templates/CS.TypeEditor/CustomControlLibrary.Protocol/Endpoints/CustomTypeEditorOKClickResponse.cs
using Microsoft.DotNet.DesignTools.Protocol.DataPipe;
using Microsoft.DotNet.DesignTools.Protocol.Endpoints;

namespace CustomControlLibrary.Protocol.Endpoints
{
    /// <summary>
    ///  Response class for this endpoint. This is not returning any relevant data, but this class is still needed
    ///  to meet the infrastructure conventions.
    /// </summary>
    public class CustomTypeEditorOKClickResponse : Response.Empty
    {
        public static new CustomTypeEditorOKClickResponse Empty { get; } = new CustomTypeEditorOKClickResponse();

        private CustomTypeEditorOKClickResponse()
        {
        }

        public CustomTypeEditorOKClickResponse(IDataPipeReader reader)
            : base(reader)
        {
        }
    }
}

File: Templates/TypeEditor/src/Templates/Templates/VB.TypeEditor/CustomControlLibrary.Protocol/Endpoints/CustomTypeEditorOKClickResponse.vb
Imports Microsoft.DotNet.DesignTools.Protocol.DataPipe
Imports Microsoft.DotNet.DesignTools.Protocol.Endpoints

Namespace Endpoints
    ''' <summary>
    '''  Response class for this endpoint. This is not returning any relevant data, but this class is still needed
    '''  to meet the infrastructure conventions.
    ''' </summary>
    Public Class CustomTypeEditorOKClickResponse
        Inherits Response.Empty

        Public Shared Shadows ReadOnly Property Empty() As New CustomTypeEditorOKClickResponse()

        Private Sub New()
        End Sub

        Public Sub New(ByVal reader As IDataPipeReader)
            MyBase.New(reader)
        End Sub
    End Class
End Namespace
